runtime.md

```diff
@@ -15,9 +15,9 @@ This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
 * **`status`**: (string) is the runtime state of the container.
 The value MAY be one of:
-    * `created`: the container has been created but the user-specified code has not yet been executed
-    * `running`: the container has been created and the user-specified code is running
-    * `stopped`: the container has been created and the user-specified code has been executed but is no longer running
+    * `created`: the container process has neither exited nor executed the user-specified code
```

Comment:
>  The container is in a created state when it has been fully initialized but is not yet running the containers main process.

#507 addresses this by adding a `creating` state, and that seems orthogonal to the stopped → exists fix which I'm focusing on here.  Until then, `created` seems like the best stand in (a container being created is certainly not `running` or `stopped`).